src/main/java/org/cascadebot/cascadebot/music/MusicHandler.java

```diff
@@ -114,7 +112,9 @@ public boolean removePlayer(long guildId) {
 
     public void purgeDisconnectedPlayers() {
         // Removes all players that are not connected to a channel unless they have supported us on Patreon
-        players.entrySet().removeIf(entry -> entry.getValue().getConnectedChannel() == null && !GuildDataManager.getGuildData(entry.getKey()).isFlagEnabled(Flag.MUSIC_SERVICES));
+        players.entrySet().removeIf(entry -> entry.getValue().getConnectedChannel() == null
+                //&& !GuildDataManager.getGuildData(entry.getKey()).isFlagEnabled(Flag.MUSIC_SERVICES)
```

Comment:
```suggestion
                //&& !GuildDataManager.getGuildData(entry.getKey()).isFlagEnabled(Flag.MUSIC_SERVICES)
```
This could be converted over to the new system fairly easily